.github/workflows/ci.yml

```diff
@@ -422,18 +422,27 @@ jobs:
         if: needs.set-condition.outputs.rebuild_parachain == 'false'
         run: |
           docker pull litentry/litentry-parachain:latest
+          docker pull litentry/litentry-chain-aio:latest
 
       - name: Save docker image
         run: |
           docker save litentry/litentry-parachain:latest | gzip > litentry-parachain-dev.tar.gz
+          docker save litentry/litentry-chain-aio:latest | gzip > litentry-chain-aio-dev.tar.gz
```

Comment:
You could save multiple images to one tgz file (if you want), so that we once need upload/download once 😂

Reply:
ok